src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -889,6 +893,8 @@ static PyMemberDef TopicPartition_members[] = {
           " :py:const:`OFFSET_STORED`,"
           " :py:const:`OFFSET_INVALID`\n"
         },
+        {"metadata", T_STRING, offsetof(TopicPartition, metadata), READONLY,
+         "attribute metadata: Metadata (string)"},
```

Comment:
missing  :

Reply:
should say bytes, not string.

And perhaps a meatier explanation:
"Optional application metadata committed with the offset"